contrib/inventory/gce_googleapiclient.py

```diff
@@ -171,6 +185,8 @@ def get_inventory(instances):
 
 
 def main(args):
+    queue_zones = Queue()
+    queue_instances = Queue()
     project = args['--project']
     all_projects = args['--all-projects']
     zone = args['--zone']
```

Comment:
Can't we just assign zone_list here? docopt will return list of zones. Additionally, "zone_names" would be better name I think